build/_build/Build.Steps.cs

```diff
@@ -339,6 +333,32 @@ partial class Build
         .DependsOn(PublishNativeProfilerLinux)
         .DependsOn(PublishNativeProfilerMacOs);
 
+    Target CreateDdTracerHome => _ => _
+       .Unlisted()
+       .After(PublishNativeProfiler, CopyIntegrationsJson, PublishManagedProfiler)
+       .Executes(() =>
+       {
+           // start by copying everything from the tracer home dir
+           CopyDirectoryRecursively(TracerHomeDirectory, DDTracerHomeDirectory, DirectoryExistsPolicy.Merge);
+
+           if (IsWin)
+           {
+               // windows already has the expected layout
+               return;
+           }
+
+           // Move the native file to the art
```

Comment:
the art?

Reply:
🤣 I have no idea what that was meant to say

Reply:
to the architecture-specific directory?

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1531/commits/2f637757b289b4d71ad448340babd2554e61e676